pkg/bootstrapper/bootstrapper.go

```diff
@@ -543,10 +543,6 @@ func (wmcb *winNodeBootstrapper) updateKubeletService(config mgr.Config, kubelet
 		return fmt.Errorf("no existing config found")
 	}
 
-	// Stop the kubelet service as there could be open file handles from kubelet.exe on the plugin files
-	if err := wmcb.kubeletSVC.stop(); err != nil {
-		return fmt.Errorf("unable to stop kubelet service: %v", err)
-	}
```

Comment:
The assumption now is that you need to call `updateKubeletService()` only after you have stopped the service. Don't you need a guard for this to be on the safe side?

Reply:
I will revert this change